base_report_docx/report/report_docx.py

```diff
@@ -64,11 +64,19 @@ def create_source_docx(self, cr, uid, ids, context=None):
 
         report = self._get_convert_file(
             tmp_folder_name, output_report[output_type])
-
+        # Put the pdf to sharefolder before delete the temp folder
+        self.save_pdf_to_share(cr, uid, ids, tmp_folder_name, context)
         self._delete_temp_folder(tmp_folder_name)
 
         return (report, output_type)
 
+    def save_pdf_to_share(cr, uid, ids, tmp_folder_name, context):
+        """
+            Override this method to save the pdf on your own server
+            path. The files created before will be deleted.
+        """
+        pass
```

Comment:
```suggestion
        raise NotImplementedError
```
The error should not be muted.